django/cantusdb_project/main_app/views/source.py

```diff
@@ -220,7 +168,13 @@ def get_queryset(self):
             )
             q_obj_filter &= indexing_search_q
 
-        return queryset.filter(q_obj_filter).distinct()
+        return (
+            queryset.filter(q_obj_filter)
+            .distinct()
```

Comment:
Hmm.. I tried removing distinct() and it didn't change the number of results so I'm not sure it's doing anything. It also reduces the runtime for loading the page so it might be best to remove this altogether.